### DISCUSSION TITLE
Live Adjustment

### DISCUSSION THREAD

**Discussion by DeNohvo** [Jyers/Marlin#183] [Q&A] [answered]
I set the live Adjustment to enabled, then when i go back into the menu it reverts back to disabled.

I also tried to use the tune option when printing, when i set the z offset I'm unable to click down to select the my value. It still let's me change the value but clicking down on the knob does not set it.

**Comment by Jyers** [comment, marked as answer]
The live adjust option resets by design, it disables when you leave the menu because it needs to rehome anytime it is enabled. Getting stuck in the value modification may be a result of some synchronization I'm doing with the gcode, I'll see if I can fix it.

**Reply by DeNohvo** [reply]
not sure i understand how to use it then if it gets disabled once you leave the menu or is it that live adjust gets enabled but the menu item resets?

**Reply by Schnabulation** [reply]
You use the live-adjust in the menu so you can live-adjust the offset while feeling for the perfect height with a piece of paper. Once the offset is at the correct height, you save the value and back out of the menu.

The life-adjust is just used for that menu. When disabled you need to re-home the Z-axis between any changes to see if it was correct.

**Reply by DeNohvo** [reply]
So i went in that menu, enabled live adjust. Used the "home Z axis" button and moved the nozzle to the center of the bed.

Once that was done i used the paper to get the nozzle close, my offset was -0.12. I then used the "save" button and everything seemed good, but when I used "home Z axis" again it was no longer scratching the paper. 

It is now printing and the first layer looks good, I guess i should have "Auto Home" after exiting the menu.

Does that seem correct?

**Reply by Jyers** [reply]
Sometimes moving the axis doesn't always move accurately, due to the nature of such small moves having very little torque. That is why it is good practice to rehome the z axis and tune you z-offset again before saving it, just to make sure it is accurate.

**Reply by Schnabulation** [reply]
> Sometimes moving the axis doesn't always move accurately, due to the nature of such small moves having very little torque.

I actually don‘t think that is the case: the Z-probe offset only takes effect on the next bed-leveling with G29.
So, what I do: I go into the Z-offset menu and do the same paper test as @JonSnow88 mentioned. Then I save the offset, back out of the menu and do a re-level. Then I go back into the menu and home Z again. It will then be perfect.
